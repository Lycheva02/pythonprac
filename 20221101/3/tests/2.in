t = Grange(840, 4, 99999)
print(t)
print(bool(t[9:9]))
print(list(t[34:678:2]))