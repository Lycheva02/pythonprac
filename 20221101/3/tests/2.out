grange(840, 4, 99999)
False
[34, 544]
